napari/_qt/_tests/test_qt_notifications.py

```diff
@@ -192,23 +212,23 @@ def test_notification_error(mock_show, monkeypatch, qtbot):
     except ValueError as e:
         notif = ErrorNotification(e)
 
-    # This test creates TracebackDialog which is not added to qtbot
-    # but its parent is same as parent of NapariQtNotification.
-    # so create dummy parent allow to not leak widget.
-    # Current parent structure, where QWidget is controled by qtbot:
-    # QWidget
-    # ├── NapariQtNotification
-    # └── TracebackDialog
-    widget = QWidget()
-    qtbot.add_widget(widget)
+    show_count = 0
+
+    def mock_show_dialog(self):
```

Comment:
Optional: might be nice to either define a fixture for this to be consistent with `NapariQtNotification`, or just use a plain function instead of a callable and pass the class for which you want to mock `show` (i.e. `NapariQtNotification` or `TracebackDialog`).